15_tictactoe/go/grid/grid_test.go

```diff
@@ -35,14 +37,36 @@ func TestIsWin(t *testing.T) {
 				Cells: [9]string{"X", "X", "X", "", "O", "X", "", "X", "X"},
 			},
 			wantIsWin:  true,
-			wantWinner: "X",
+			wantWinner: humanWin,
 		},
 		{
 			g: Grid{
 				Cells: [9]string{},
 			},
 			wantIsWin:  false,
-			wantWinner: ""},
+			wantWinner: computerWin,
+		},
+		{
+			g: Grid{
+				Cells: [9]string{"O", " ", "X", "O", " ", "X", "O", "X", " "},
+			},
+			wantIsWin:  true,
+			wantWinner: computerWin,
+		},
```

Comment:
I don't think this new test passes

```suggestion
		{
			g: Grid{
				Cells: [9]string{"O", " ", "X", "O", " ", "X", "O", "X", " "},
			},
			wantIsWin:  true,
			wantWinner: computerWin,
		},
		{
			g: Grid{
				Cells: [9]string{
				"O", " ", "X", 
				"O", "O", "X", 
				"X", "X", "O",
				},
			},
			wantIsWin:  true,
			wantWinner: computerWin,
		},
```

Reply:
Added some and have high test coverage again.